pkg/operatorsource/handler.go

```diff
@@ -4,15 +4,23 @@ import (
 	"context"
 
 	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/shared"
-	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	v1 "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
```

Comment:
same comment as above

Reply:
This still needs to be addressed in all places.